src/Data/Functor/Alt.hs

```diff
@@ -321,3 +337,21 @@ instance Alt Monoid.First where
 
 instance Alt Monoid.Last where
   (<!>) = mappend
+
+-- Helpers
+
+-- Required to work around https://hub.darcs.net/ross/transformers/issue/67
+
+#if MIN_VERSION_transformers(0,5,6)
```

Comment:
Can you define these in an internal module? That way, we won't have to redefine them in multiple locations.